packages/components/src/BaseButton/BaseButton.styles.js

```diff
@@ -155,12 +157,14 @@ export const LoadingOverlay = css`
 export const subtle = css`
 	border-color: ${ui.get('controlBorderColorSubtle')};
 	color: ${ui.color.text};
+	fill: ${ui.color.text};
```

Comment:
I wish we didn't have to do this 🙈 
Maybe an enhancement we can make in the future